utils/oscap-cvrf.c

```diff
@@ -0,0 +1,174 @@
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+/* Standard header files */
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <getopt.h>
+#include <assert.h>
+#include <math.h>
+
+#include <oscap.h>
+#include <oscap_error.h>
+#include <cvrf.h>
+#include <oscap_source.h>
+
+#include "oscap-tool.h"
+
+static bool getopt_cvrf(int argc, char **argv, struct oscap_action *action);
+static int app_cvrf_evaluate(const struct oscap_action *action);
+static int app_cvrf_export(const struct oscap_action *action);
+static int app_cvrf_validate(const struct oscap_action *action);
+
+static struct oscap_module* CVRF_SUBMODULES[];
+
+struct oscap_module OSCAP_CVRF_MODULE = {
+	.name = "cvrf",
+	.parent = &OSCAP_ROOT_MODULE,
+	.summary = "Common Vulnerability Reporting Framework",
+	.submodules = CVRF_SUBMODULES
+};
+
+static struct oscap_module CVRF_EVALUATE_MODULE = {
+	.name = "evaluate",
+	.parent = &OSCAP_CVRF_MODULE,
+	.summary = "Evaluate system for vulnerabilities",
+	.usage = "[options] <cvrf file> results-file.xml",
```

Comment:
Is the name of results file hardcoded to `results-file.xml`? If it isn't and user can specify it, I think it should follow the same convention as `<cvrf file>`.